src/Console/Command.php

```diff
@@ -52,12 +68,20 @@ public function __construct()
      * Set the name this command uses in the collection.
      *
      * Generally invoked by the CommandCollection when the command is added.
+     * Required to have at least one space in the name so that the root
+     * command can be calculated.
      *
      * @param string $name The name the command uses in the collection.
      * @return $this;
+     * @throws \InvalidArgumentException
      */
     public function setName($name)
     {
+        if (strpos($name, ' ') === false) {
```

Comment:
This will allow space to be a first character. Is that something we should worry about?

Reply:
Yes. I'll fix that.